### PR TITLE
Ssh command fix

### PR DESCRIPTION
**Pull request by robert-sandor**
This fixes an issue on cloud_slang_content, where ssh_command could not be run on certain devices.
It enables a ChannelShell mode for the operation, so the commands are issued just like they would be from a terminal with ssh.

### CODE REVIEW

File: cs-ssh/src/main/java/io/cloudslang/content/ssh/services/impl/SSHServiceImpl.java [hunk 1]
```diff
@@ -10,28 +10,17 @@
...
 import io.cloudslang.content.ssh.utils.CacheUtils;
 import io.cloudslang.content.ssh.utils.IdentityKeyUtils;
 import io.cloudslang.content.utils.StringUtilities;
+import org.apache.commons.io.IOUtils;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.io.UnsupportedEncodingException;
+import java.io.*;
```
Comment (Bonczidai) [inline cs-ssh/src/main/java/io/cloudslang/content/ssh/services/impl/SSHServiceImpl.java]: I no like star imports

File: cs-ssh/src/main/java/io/cloudslang/content/ssh/services/impl/SSHServiceImpl.java [hunk 2]
```diff
@@ -142,6 +134,52 @@ public SSHServiceImpl(ConnectionDetails details, IdentityKey identityKey, KnownH
...
+            channelShell.connect(connectTimeout);
+
+            final PrintWriter printWriter = new PrintWriter(new OutputStreamWriter(shellIn, characterSet));
+
+            printWriter.println(command);
+            printWriter.flush();
+
+            try {
+                Thread.sleep(commandTimeout);
+            } catch (InterruptedException ignored) {
```
Comment (Bonczidai) [inline cs-ssh/src/main/java/io/cloudslang/content/ssh/services/impl/SSHServiceImpl.java]: hm, maybe logging

File: cs-ssh/src/main/java/io/cloudslang/content/ssh/services/impl/SSHServiceImpl.java [hunk 3]
```diff
@@ -142,6 +134,52 @@ public SSHServiceImpl(ConnectionDetails details, IdentityKey identityKey, KnownH
...
+
+            channelShell.connect(connectTimeout);
+
+            final PrintWriter printWriter = new PrintWriter(new OutputStreamWriter(shellIn, characterSet));
+
+            printWriter.println(command);
+            printWriter.flush();
+
+            try {
+                Thread.sleep(commandTimeout);
```
Comment (Bonczidai) [inline cs-ssh/src/main/java/io/cloudslang/content/ssh/services/impl/SSHServiceImpl.java]: do you need to wait here without a condition?

Comment (robert-sandor) [inline cs-ssh/src/main/java/io/cloudslang/content/ssh/services/impl/SSHServiceImpl.java]: yes, didn't find a reliable way to check when i receive the full output

Comment (Bonczidai) [inline cs-ssh/src/main/java/io/cloudslang/content/ssh/services/impl/SSHServiceImpl.java]: f94a2e540ac512fa4ee29cefe32a0739_alrighty-then-meme-alrighty-then_625-351

**Pull request merged**